### PR TITLE
Add instructions to help future Wunderer deploy Battle Pets API

### PR DESCRIPTION
**Pull request by brandonecraig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +147,22 @@ $ export JDBC_DATABASE_URL="jdbc:postgresql://localhost:5432/pets_test?user=[use
 $ sbt test
 ```
 
+## Deployment
+
+### Heroku ENV Variables
+
+The follow are required to deploy the pet service to Heroku.
```
Comment (bhb) [inline README.md]: Minor thing, but I misread this as "you need these env vars locally in order to successfully deploy code to heroku". Maybe rephrase to something like "The follow environment variables must be present in Heroku for the app to run" or something. Or honestly, just "In Heroku, set the following 'Config Vars'"

**Review by bhb — APPROVED** [review]
Nice!

File: README.md (hunk 1 above)
Comment (brandonecraig) [reply README.md]: Solid feedback. I'll clean that up.
